### ISSUE TITLE
Need help on installation

### ISSUE THREAD

**Issue by richjr93** [saleem-hadad/finance#54]
How can I install this in my localhost and web hosting server?

I tried following the installation instructions but the command seems not to be working. I tried installing docker but it doesn't work either. I have no idea how to do it. FYI, I am not a programmer, hence I have little to zero knowledge. I just really wanted to try this.

I would appreciate it if anyone here could guide me in detail on how to install it. Thank you very much!

**Comment by dreyerm** [comment]
Same here, can't get this installed. Tried both with docker, and manually with composer.

Docker keeps throwing the following error:

`docker run --rm \
    -u "$(id -u):$(id -g)" \
    -v $(pwd):/var/www/html \
    -w /var/www/html \
    laravelsail/php81-composer:latest \
    composer install --ignore-platform-reqs
    
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "proc" to rootfs at "/proc": mount proc:/proc (via /proc/self/fd/6), flags: 0xe: permission denied: unknown.`

I'm trying to run this inside a proxmox LXC container, but I checked and nested virtualisation is enabled so that shouldn't be the problem, although it looks to be some sort of permission issue, sudo-ing gives the same error. I also tried setting folder permissions to 777 which didn't solve it.

Using the composer installer throws the following error:

`composer install
PHP Parse error:  syntax error, unexpected '|', expecting variable (T_VARIABLE) in /usr/share/php/Composer/IO/BaseIO.php on line 163`

Any ideas on what's wrong here? I'd love to use the app! It looks impressive and useful.

Thanks!
